imports/lib/schemas/discord_cache.ts

```diff
@@ -0,0 +1,38 @@
+import * as t from 'io-ts';
+import { date } from 'io-ts-types';
+import { buildSchema, Overrides } from './typedSchemas';
+
+// Don't use the BaseCodec here - unlike most database objects, this isn't
+// manipulated by users, so many of the fields don't make sense
+export const DiscordCacheCodec = t.type({
+  _id: t.string,
+  createdAt: date,
+  snowflake: t.string,
+  type: t.string,
+  object: t.object,
+});
+
+const DiscordCacheOverrides: Overrides<t.TypeOf<typeof DiscordCacheCodec>> = {
+  createdAt: {
```

Comment:
Any reason to omit (or include) an `updatedAt`?

Reply:
No strong reason, I can add one.